htdocs/web_portal/views/site/edited_api_auth.php

```diff
@@ -1,6 +1,16 @@
 <div class="rightPageContainer">
     <h1 class="Success">Success</h1><br />
-    The API authenication credential has now been updated. Type:<?php xecho($params['apiAuthenticationEntity']->getType()) ?>, identifier: <?php xecho($params['apiAuthenticationEntity']->getIdentifier()) ?>.
+    The API authenication credential has now been
+    <?php if ($params['isRenewalRequest']) {
+        echo 'renewed for';
```

Comment:
I'd mirror the behaviour in the edit case here

```suggestion
        echo 'renewed, ';
```

so the output would be

`The API authenication credential has now been renewed, identifier: ABC. `

Reply:
Done. replicated the same as edit.